job-templates/kubernetes_release.json

```diff
@@ -33,7 +34,12 @@
     ],
     "windowsProfile": {
       "adminUsername": "azureuser",
-      "adminPassword": "replacepassword1234$"
+      "adminPassword": "replacepassword1234$",
+      "windowsPublisher": "MicrosoftWindowsServer",
+      "windowsOffer": "WindowsServerSemiAnnual",
+      "windowsSku": "Datacenter-Core-1809-with-Containers-smalldisk",
+      "imageVersion": "1809.0.20190214",
+      "windowsDockerVersion": "18.09.0"
```

Comment:
the latest is 18.09.02. I just downloaded this file, and started a test with 18.09.2 to make sure it deploys.

Reply:
This Docker version was what aks-engine 0.31.1 defaulted to. If 18.09.02 works, we go with that.

Reply:
ok, will submit a fix there too https://github.com/Azure/aks-engine/issues/547